persistence-api/src/main/java/io/stargate/db/query/builder/BuiltDML.java

```diff
@@ -278,7 +268,13 @@ private void handleTimestamp() {
     private void handleConditions() {
       for (BuiltCondition bc : dml.data.conditions) {
         Condition.LHS lhs = createLHS(bc.lhs());
-        TypedValue v = handleValue(lhs.toString(), lhs.valueType(), bc.value());
+        TypedValue v;
+        if (bc.predicate().equals(Predicate.IN)) {
```

Comment:
We had a bug for `Predicate.IN`, that I've reproduced in this unit test:
https://github.com/stargate/stargate/pull/958/files#diff-f4326ae765fc08166ff59d0f5c07f7c2478f35032e4aa1efd74f8e47b77249baR202
It is fixed now